### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by likemusic**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Fix broken internal links detected by [Xenu](http://home.snafu.de/tilman/xenulink.html).
| Fixed ticket? | No.

This PR contains:
- Fix for one image filename;
- Fix one link in md-file;
- Converted 2 schemes to `xml` format (from `drawio`);

# Xenu screenshots
For master-branch:
image

For PR-branch:
image

Remains `/images/preston-logo_2x.png` url is fase-positive, and should be ignored. `style.css` in fact contains `/images/preston-logo@2x.png`. Seems that Xenu doesn't support `@` char in urls, that leads to renaming file to not exists `preston-logo_2x.png`.

### CODE REVIEW

File: src/static/schemas/1.7/create-resource.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...sKRAM4YVMKUpWxEtXNUDDsb/mK7LBn/Eqg43UsrWQK00KOKRdpRlwoKYqbtpI542Ir94p8jAdF3abMMrk8jFLD7QSnC0LEMiJ5rDHBxQRKrH9F+YBipFSKyCbYMYxomQRg+CDJdKj66ZR1tpgLvCuolKm+oAZXBvfQxfVag8UbJRb2Y6StxWQ5hgNK/i0c7wh5RiLYu4SHPCi8NERK5algUWDScW4CSOxSNd3xz33/gAtjIuQLViMaBUvpQ2N67RhqwN1Nqrj1mw6ajKpbtFCeVKLDjSDfsXeCvMNAYd9iwA/PwL03auKAEMNL18ALW9IuQKkFCj4YaTY50DK6C1VtHrPuVPFeZK/rVl0+jj7U5J+JNAx48rzEuDad+rKPCYEi6ABx8GdJO9SR5m/TFWwz2+guDFurVz+W5ry1nCGz5qzo/Hwjgi5iHmbS/kSrpLvdwo+qbCvCBUIZjq5w7J7Ku2r0uGAVl9fsTX38XF/EogvcBu+ch/DQa3a0eHFMUUC4mRt+Sa0qKFT6Z4lLF2zHmqcQ8BlJ1GjDjBXbOOFMHT/S4ElV0JZaKRPc8ghlFZ6PqRPdzRLuWmhs/CRUT0kOf1uMeksOcbqa1iYoHQI+Mi7Nae/pd8R5O7aWe1rA5VRi1KnilAvCxK5LxyPEu5losTIbiEkl4gSemEDnDBh8QqkO/ndhqJ4Ka/TD3H06uzFEgxN4wCtwoJTQrMy6LCScfIskuWbkfuT8eJeUVY5xItj9utTZLs8BV6my3WwN75/ftqMxnj65D3+Y/1xY2qQ+fbpIygmHMOxWCx34vuMByRe5HEloz5wch/DwQO5hOyXEaOWoihmAj9f7ZQfFh+z+kXpA8SXjzCKiH1GWSSEVLUDF6Ylk/kc932/KSEFg5FnHCUk3dOCZR6khWFDWnAb8sLgBHmh0ZodPlVVHFg5W82BK8Zo9N4qU5Rnq3BFx24hi53c9Bg6n3XCyoU33Xeue6WvHn6YdPvdYrs2UUEd84kOqeSZnV4vat7Y5CXZZP8ARvbF2GQjHPRfQxShREHw7hdkk9XEf8w/zp37HasFCKfnio3H1IvKgi7+P6jhVRl/ZN66F80Ceh1ZrxZEiNNjqhuDYAQbgyICY8n2fEkQJfn7ZZFybgSYbh0B/Y4sQIeS+cxEL44jIJa/0mfdy3+FsN//Cw==</diagram></mxfile>
```
Comment (NeOMakinG) [inline src/static/schemas/1.7/create-resource.xml:2]: Could you add new line at the end of these xml files ? Thanks :)

File: src/static/schemas/1.7/update-resource.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
...IubLpTpBHYD4PiLB08Sj8yZFownKbEBTF2ElG1QTCwN2wGG/hE8QpbSRKN1yONMvpkFSVbgBPdZXvtJb0G2BXPRPkAJnmfptRQuXwIU0WFHNG1zkJZEdLMccHFGAiuf0FmIdCpGBFZF24cYoIXoXCcIUnkqU3XaO8tQXGYV+ClKveAxXbxg6iiio1R4o2KqxMS9m7EkkzjvolfpoZ35AKjFXed0EO8aD40ZMrsm2NLA2alJwbURzyZHx7OrDva2yhjPt0RUNEynwpfKhdpw87A6i3Uy274tNJm0ubHs3Bk3p01HDoV3BiYFssAvYtA/z4DDC0ryoDjBt8+SzY8saUK2BKzoLvZop5DqZM3o6Kzug591FxnsPfbHh0/rj4S4p+xNEx58r1YqG179SWOZRzGogCCL07Kd4lRqi7TiAxz28CuNFujcz+W7ryVrPGz7qzp/Ngj7kcRL/NrGwIW9n3e0WfxDiUjBIFU0zOsKieWIeyVW/QGesx3TAXjscTR2wFXfzKYgy8ym2nSS8GBHGRJyvDt7FFNZ3L8CxoaevVVGPVCZeuRLWqcS6fxgtpaP9MiSUDxbVQSz7tKQcTUqr5kHz6s1nabQOdRY9MqinJGvbLSWc5Y4xhgwszlDQRMfLb78lLBDm1bkn72iylVVLUqdLTyzJEFgjHU4R9mRQxMTvUyCVSRPNWIwRhRMP41QSgEYiiqYdiPxePolg5b1w6WrLjIj1YJvYP5oZ9RcdHnRuWPqx2kc7yFNyYrzfeQfvn09N2MoX5k/P4r/Hnjd6gx7ePHwTwORJKBmI5EdelzMPhKkshqcQRC3dBrNuTI4TiKxVAHZefkHJ4/lZTvEB8TO8pCvcQWz+KVpgfUmkiGaRuNWK/GofGcglD1207eLzRxNGOCo/+6d/Qa+l/3JL+7Zb8PzpB/m91Zo9XUqX4VbFWid+SM1qDt6wI5dpKmtAyO0Rhryg9Rs5nY7C04W37nWGvDNX6C0h72C+NNzrKJWLWUV0ynjnmm5eXN9V4SdU4rNHIvJhqbKVD81cPJRzBw7+kciwf/McC5Nxnv2V0MOH0urB1mc3b4/9MGl6V8yf6rX3RY6B5Yczdb97JMPEhWabaMZGNkNSDDoCUewH18BJL+ffLUuXcFNDtKgWGPXVAk0v6Mx29OJEIs/g9Pq1e/NOD+e4/</diagram></mxfile>
```
Comment (NeOMakinG) [inline src/static/schemas/1.7/update-resource.xml:2]: Same here

**Review by NeOMakinG — CHANGES_REQUESTED** [review]

**Comment by NeOMakinG** [comment]
Also, there are some conflicts that needs to be resolved!

Can we reproduce all these broken links in production ? Could you sen me an uri where we can verify this ? (To be sure it's not related to a development environnement at all !

Thanks for all the great work you do on the doc these days, this is pretty cool :)

**Comment by likemusic** [comment]
I have added line breaks to the end of all files in the `src/static/schemas/1.7`.

**Comment by likemusic** [comment]
> Could you send me an uri where we can verify this ?

At the moment only this errors remains:
image

## 1
https://devdocs.prestashop.com/1.7/development/components/form/types-reference/date-range/ references to https://devdocs.prestashop.com/1.7/development/components/form/types-reference/img/date_range.png

image

## 2
https://devdocs.prestashop.com/1.7/development/components/form/types-reference/material-multiple-choice-table/ referenses to https://devdocs.prestashop.com/1.7/development/components/form/types-reference/material-multiple-choice-table/material-choice-table

image

## 3
https://devdocs.prestashop.com/1.7/development/components/grid/tutorials/extend-grid-with-javascript/ references to https://devdocs.prestashop.com/1.7/development/components/grid).

image

File: src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ add or remove grid feature while loading only necessary javascript code. In this
 
 ## Getting started
 To start we will need to have a plain grid already created in php side
-(you can find more docs about grid here https://devdocs.prestashop.com/1.7/development/components/grid).
+(you can find more docs about grid here in [The Grid component page]({{< ref "/1.7/development/components/grid/" >}}).
```
Comment (atomiix) [inline src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md:18]:
```suggestion
(you can find more docs about grid in [The Grid component page]({{< ref "/1.7/development/components/grid/" >}}).
```

**Review by atomiix — APPROVED** [review]

**Comment by likemusic** [comment]
@NeOMakinG I have alrady merge conflicts and done all yours changes requests.

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Thanks @likemusic

**Pull request merged**